bot/seasons/halloween/hacktober-issue-finder.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+import logging
+import random
+from typing import Dict, Optional
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+URL = "https://api.github.com/search/issues?per_page=100&q=is:issue+label:hacktoberfest+language:python+state:open"
+HEADERS = {"Accept": "application / vnd.github.v3 + json"}
+
+
+class HacktoberIssues(commands.Cog):
+    """Find a random hacktober python issue on GitHub."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.cache_normal = None
+        self.cache_timer_normal = datetime.datetime(1, 1, 1)
+        self.cache_beginner = None
+        self.cache_timer_beginner = datetime.datetime(1, 1, 1)
+
+    @commands.command()
+    async def hacktoberissues(self, ctx: commands.Context, option: str = "") -> None:
+        """Get a random python hacktober issue from Github."""
```

Comment:
Since there is also a `beginner` option, I'd like to see that documented as well so it shows up in `.help`.